internal/pkg/skuba/deployments/os.go

```diff
@@ -29,7 +29,7 @@ const (
 	SUSEOSID = "suse"
 )
 
-func (t *Target) osRelease() (map[string]string, error) {
+func (t *Target) OSRelease() (map[string]string, error) {
```

Comment:
let's not change the interface if it's not necessary. Can you remove this part of your commit?

Reply:
True. Done.